src/Scriban.Tests/TestRuntime.cs

```diff
@@ -370,20 +378,21 @@ public void TestJson()
             var expected =
                 @"
 [
-  { 
```

Comment:
Spaces have been modified while they should not

Reply:
Done.